docs/source/library/SynthesisResults.rst

```diff
@@ -0,0 +1,36 @@
+Results
+=======
```

Comment:
This should probably be called "Synthesis Results" so that the menu entries for mapping and synthesis have different names.